src/Tribe/Repositories/Event.php

```diff
@@ -1641,10 +1656,12 @@ protected function order_by_organizer( $after = false ) {
 		);
 
 		$filter_id = 'order_by_organizer';
+		$order     = $order === null
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 259bf1967793249d0ae8ae50994157b7</sub>